corehq/apps/linked_domain/util.py

```diff
@@ -3,14 +3,29 @@
 from couchdbkit import ResourceNotFound
 
 from corehq import toggles
+from corehq.apps.accounting.utils import domain_has_privilege
 from corehq.apps.app_manager.dbaccessors import get_app
 from corehq.apps.app_manager.exceptions import MultimediaMissingError
 from corehq.apps.hqmedia.models import CommCareMultimedia
 from corehq.apps.hqwebapp.tasks import send_html_email_async
 from corehq.apps.linked_domain.remote_accessors import fetch_remote_media
+from corehq.privileges import RELEASE_MANAGEMENT
 from corehq.util.timezones.conversions import ServerTime
 
 
+def can_access_linked_domains(user, domain):
```

Comment:
new decorator(s), new tests? :)

would be nice to have tests for these directly